### ISSUE TITLE
9.11 additional functions clash

### ISSUE THREAD

**Issue by ibell** [jowr/librefprop.so#31]
In version 9.11 of REFPROP, the additional functions that are included in ``src/PASS_FTN_ALT.f`` are no longer needed since they are provided in REFPROP itself.  Not sure how best to fix this, but I "fixed" it by commenting out these functions in ``src/PASS_FTN_ALT.f``.  Fun times.

**Comment by jowr** [comment]
Shouldn't we just rermove `src/PASS_FTN_ALT.f` from the sources?

**Comment by ibell** [comment]
Yes that would probably do it, but only if the REFPROP version is >=9.11

**Comment by speredenn** [comment]
Excluding it from the compilation process using the makefile would be better, I think, if the file is still distributed with REFPROP 9.1.1. Doing so, you don't delete it, and you don't need to comment the lines out.

**Comment by jowr** [comment]
Oh, that is what I meant: Remove it from the list of files to compile - not deleting it.

**Comment by speredenn** [comment]
OK ! :-) ; Does somebody have a REFPROP 9.11 to test the code after the makefile has been modified? It seems that Ian has it. Do you also have a version at hand, Jorrit? Personally, I don't, but I can make the modification and make a pull request, as long as one of you test the code after that :-).
